docs/Tutorials/DomainCreation.md

```diff
@@ -0,0 +1,215 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Domain Creation {#tutorial_domain_creation}
+
+### Domain Decomposition
+The spatial domain on which the solution is computed is divided into
+non-overlapping cells. For 2D and 3D domains, we use curvilinear
+quadrilateral and curvilinear hexahedral elements, respectively. The
+specification of a particular spatial domain is given by the DomainCreator.
+DomainCreators specify the properties of the initial elements (before AMR takes
+place), which are then used by Domain to create Blocks. Each Block
+holds a CoordinateMap as well as information about its neighbors. For
```

Comment:
How about "...a CoordinateMap (mapping from a logical cube that goes from [-1,1] in each dimension to the curvilinear hexahedral element that the block describes) as well as..."